src/database/migrations/0023-insert-system-user.sql

```diff
@@ -0,0 +1,11 @@
+INSERT INTO users ( authentication_id ) VALUES ( '' );
```

Comment:
I want to call this out for discussion -- the idea here is we will always have a "system" user with no authentication id.

Our user middleware will not populate a user if `sub` is `''` which I think makes this a particularly safe choice (if we had the system user as something like `system` as the authenticaftion ID then in theory someone could figure out a way to make a jwt with that as the subject and become the system user.  In this case even if they did manage to create a valid jwt with a blank sub, our middleware would treat that as an invalid user.